src/Avalonia.Themes.Fluent/Controls/CalendarDatePicker.xaml

```diff
@@ -19,125 +19,131 @@
   <Styles.Resources>
     <sys:Double x:Key="CalendarDatePickerCurrentDayFontSize">12</sys:Double>
   </Styles.Resources>
-  
+
   <Style Selector="CalendarDatePicker">
 
-    <Setter Property="Background" Value="{DynamicResource CalendarDatePickerBackground}"/>
-    <Setter Property="Foreground" Value="{DynamicResource CalendarDatePickerForeground}"/>
-    <Setter Property="BorderBrush" Value="{DynamicResource CalendarDatePickerBorderBrush}"/>
-    <Setter Property="BorderThickness" Value="{DynamicResource CalendarDatePickerBorderThemeThickness}"/>
+    <Setter Property="Background" Value="{DynamicResource CalendarDatePickerBackground}" />
+    <Setter Property="Foreground" Value="{DynamicResource CalendarDatePickerForeground}" />
+    <Setter Property="BorderBrush" Value="{DynamicResource CalendarDatePickerBorderBrush}" />
+    <Setter Property="BorderThickness" Value="{DynamicResource CalendarDatePickerBorderThemeThickness}" />
     <Setter Property="CornerRadius" Value="{DynamicResource ControlCornerRadius}" />
     <Setter Property="HorizontalAlignment" Value="Left" />
     <Setter Property="VerticalAlignment" Value="Center" />
-    <Setter Property="Padding" Value="4"/>
-    
+    <Setter Property="Padding" Value="4" />
```

Comment:
Please revert unrelated whitespace changes here also.

Reply:
I force pushed a fixed commit for this.